### DISCUSSION TITLE
Extended support for skin element animation

### DISCUSSION THREAD

**Discussion by RenovaDeus** [ppy/osu#13706] [Ideas] [answered]
Decided to make a discussion rather than a feature request for this, since it needs to be elaborated on more before it can be turned into an FR in `osu\issues`, but on peppy's [livestream](https://www.twitch.tv/ppy) during the June 30th team meeting, I asked him in [chat](https://i.imgur.com/LD3SKu5.png) whether or not he was potentially interested in additional/extended support for skin elements being animated.

I had originally suggested it through loading GIF files, and while he's not interested in utilizing GIFs (and possibly WEBMs by extension of that), he *is* interested in the extended animation bit.

The reason I'm starting the discussion is that a few things have to be established:

- For what elements should this additional animation support apply? Does it apply to elements in the ranking panel, the carousel, etc?

- What methods are preferable to use to load animated skin elements efficiently?

- How much time would it realistically take to decide and implement this?

- Is it mildly important, or can it be shelved to the backburner?

**Comment by peppy** [comment, marked as answer]
Every skin element will support it. It'll be part of the newer skinning system as it comes online. No further details are available until that happens.
